### PR TITLE
Icons8

### PR DESCRIPTION
**Pull request by yug2005**
## Description

**Current Features** : 

* Search for icons by name. 
* Filter the icons by the common styles. 
* Recolor the colorable icons. 
* View details about the icon such as style, category, and tags.
* Configure the download settings such as file name, format, and size. 
* Download SVG and image formats including PNG, JPG, and WEBP. 
* Pin the icons and view recent icons. 

Feel free to suggest other features. 

## Screencast

Search Icons
Categories
Icon Actions
Icon Detail
Configure Icon
Recolor Icon

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by danielsequeira** [comment]
Hi @yug2005,

Thank you so much for your contribution.

I just tested the extension and noticed that for some symbols the "Open in Browser" action doesn't do anything. Like for example for the "Smiling Sun" one from the "Cloud" collection. And I guess it should open this url: https://icons8.com/icon/7UYg9itOH2r7/smiling-sun no?

File: extensions/icons8/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<div align="center">
+  <img src="./media/Icons8-Art.svg" style="width: 96px; border-radius: 32px;"/>
```
Comment (danielsequeira) [inline extensions/icons8/README.md:2]: we current have a bug that doesn't allow us to render images in the README with relative paths. you will need to use the github absolute path here in order for this to work. also you could delete the media folder and have everything in assets.

File: extensions/icons8/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<div align="center">
+  <img src="extensions/icons8/assets/Icons8-Art.svg" style="width: 96px;"/>
```
Comment (danielsequeira) [inline extensions/icons8/README.md:2]: sorry, by absolute path I meant the actual link: https://raw.githubusercontent.com/raycast/extensions/1ae0136661beca0df6f2e924707e07f0d92f1800/extensions/icons8/assets/Icons8-Art.svg

File: extensions/icons8/README.md (hunk 1 above)
Comment (yug2005) [reply extensions/icons8/README.md]: Hey I couldn't get it to work. How do you get the GitHub absolute path?

File: extensions/icons8/README.md (hunk 2 above)
Comment (yug2005) [reply extensions/icons8/README.md:2]: Oh I see. Thank you

Comment (danielsequeira) [reply extensions/icons8/README.md]: I just did it 🙂

**Pull request merged**